__tests__/index.spec.ts

```diff
@@ -26,6 +26,23 @@ describe('Preserve', () => {
     expect(item.get()).toEqual(JSON.parse(expected));
   });
 
+  it('should be able to update data if data is boolean false', () => {
```

Comment:
Just some nitpick, but the text is slightly confusing. 

Maybe have something like this instead?:

Should be able to update data if data is pure boolean values